src/gl_limits.rs

```diff
@@ -0,0 +1,15 @@
+use gleam::gl::types::GLint;
+use gleam::gl;
+
+#[derive(Deserialize, Serialize)]
```

Comment:
Probably want to derive `Clone` too here, to send it around without calling `detect()` again.